Automattic/jetpack
# Boost: Add UI for recommendations to ISA

Fixes https://github.com/Automattic/boost-cloud/issues/314

## Proposed changes:

* Add two recommendation elements to ISA UI:
  * <details><summary>Groups summary</summary>
    <img width="806" alt="image" src="https://github.com/Automattic/jetpack/assets/11799079/a3b00029-57bb-4f0d-973b-77926a8ebbc3">
    </details>
  * <details><summary>Detailed report page</summary>
    <img width="769" alt="image" src="https://github.com/Automattic/jetpack/assets/11799079/eba37c17-963c-4c7e-b7ab-468e8ad00eba">
    </details>

**Note**: Looking at this after creating the PR - maybe the one from the groups summary should be removed?

### Other information:


## Jetpack product discussion



n/a

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup Boost ISA;
* Generate a report;
* Check the Summary UI and detailed report page to make sure the recommendation looks ok.

* Enable `Image CDN`;
* Make sure the recommendation disappears from both places.
